apps/prairielearn/src/pages/publicAssessmentQuestions/publicAssessmentQuestions.html.ts

```diff
@@ -0,0 +1,127 @@
+import { html } from '@prairielearn/html';
+
+import { AssessmentBadge } from '../../components/AssessmentBadge.html.js';
+import {
+  AssessmentQuestionHeaders,
+  AssessmentQuestionNumber,
+} from '../../components/AssessmentQuestions.html.js';
+import { HeadContents } from '../../components/HeadContents.html.js';
+import { Navbar } from '../../components/Navbar.html.js';
+import { TagBadgeList } from '../../components/TagBadge.html.js';
+import { TopicBadge } from '../../components/TopicBadge.html.js';
+import { type Assessment, type Course } from '../../lib/db-types.js';
+import { type AssessmentQuestionRow } from '../../models/assessment-question.js';
+
+export function PublicAssessmentQuestions({
+  resLocals,
+  assessment,
+  course,
+  course_instance_id,
+  questions,
+}: {
+  resLocals: Record<string, any>;
+  assessment: Assessment;
+  course: Course;
+  course_instance_id: string;
+  questions: AssessmentQuestionRow[];
+}) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${HeadContents({ resLocals })}
+      </head>
+      <body>
+        ${Navbar({ resLocals })}
```

Comment:
Do we need tabs here? It's not clear to me that they add anything, especially since there's only one (at least for now):

<img width="931" alt="Screenshot 2025-04-10 at 09 59 44" src="https://github.com/user-attachments/assets/f242ad6b-f229-415a-9a19-6036b1d2cfb5" />

Reply:
If you must have the tab here, please fix its link, it currently 404s:

<img width="563" alt="Screenshot 2025-04-10 at 10 00 37" src="https://github.com/user-attachments/assets/3772d518-dd82-4253-a4a3-cba29bf5f9e9" />

Reply:
Ya, we want to have the 'Files' tab their eventually too, but I'll just remove for now and worry about it when we need to. 

remove in c4cf8ff6d